conf/awsbatch.config

```diff
@@ -0,0 +1,14 @@
+//Nextflow config file for running on AWS batch
+
+params {
+  config_profile_name = 'AWSBATCH'
+  config_profile_description = 'AWSBATCH Cloud Profile'
+  config_profile_contact = 'Alexander Peltzer (@apeltzer)'
+  config_profile_url = 'https://aws.amazon.com/batch/'
+}
+
+aws.region = params.awsregion
+process.executor = 'awsbatch'
+process.queue = params.awsqueue
+executor.awscli = '/home/ec2-user/miniconda/bin/aws'
+params.tracedir = './'
```

Comment:
Any reason not to group this together with the other `params` at the top?

Reply:
None, I just copied the config file we had on sarek